lib/index.js

```diff
@@ -20,13 +20,15 @@ debug('[global] binName:', binName);
 /**
  * @param {string} baseUrl
  * @param {string} hugoVersion
+ * @param {boolean} isWithDeploy
  */
-function extendedBin(baseUrl, hugoVersion) {
+function extendedBin(baseUrl, hugoVersion, isWithDeploy) {
   debug('[extendedBin] baseUrl:', baseUrl);
   debug('[extendedBin] hugoVersion:', hugoVersion);
+  debug('[extendedBin] isWithDeploy:', isWithDeploy);
 
   const baseName = `hugo_${hugoVersion}`;
-  const baseNameExtended = `hugo_extended_${hugoVersion}`;
+  const baseNameExtended = (isWithDeploy) ? `hugo_extended_withdeploy_${hugoVersion}` : `hugo_extended_${hugoVersion}`;
```

Comment:
the parentheses are redundant here

```suggestion
  const baseNameExtended = isWithDeploy ? `hugo_extended_withdeploy_${hugoVersion}` : `hugo_extended_${hugoVersion}`;
```